File: src/main/java/ar/edu/unlam/tallerweb1/delivery/controllers/GoToAddFriend.java
package ar.edu.unlam.tallerweb1.delivery.controllers;

import ar.edu.unlam.tallerweb1.delivery.Controller;

import javax.servlet.http.HttpServletRequest;

public class GoToAddFriend implements Controller {
    @Override
    public String invoke(HttpServletRequest request) {
        String user = request.getParameter("user");
        return
                "<div>" +
                    "<form id=\"add-friend\" action=\"add-friend\" method=\"POST\"><br>" +
                        "<input id=\"friend\" name=\"friend\" type=\"text\" value=\"\"/><br><br>" +
                        "<input id=\"user\" name=\"user\" type=\"hidden\" value=\""+user+"\"/>" +
                        "<button Type=\"Submit\"/>Agregar Amigo</button>\n" +
                    "</form>\n" +
                "</div>";
    }
}

File: src/main/java/ar/edu/unlam/tallerweb1/delivery/controllers/GoToAddFriend.kt
package ar.edu.unlam.tallerweb1.delivery.controllers

import ar.edu.unlam.tallerweb1.delivery.Controller
import javax.servlet.http.HttpServletRequest

class GoToAddFriend : Controller {
    override fun invoke(request: HttpServletRequest) =
        """<div>
            <form id="add-friend" action="add-friend" method="POST"><br><
                input id="friend" name="friend" type="text" value=""/><br><br>
                <input id="user" name="user" type="hidden" value="${request.getParameter("user")}"/>
                <button Type="Submit"/>Agregar Amigo</button>
            </form>
        </div>"""

}
